### PR TITLE
Add net.sourceforge.squirrel_sql

### PR DESCRIPTION
**Pull request by ender672**
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: https://sourceforge.net/p/squirrel-sql/mailman/squirrel-sql-develop/?viewmonth=202102**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build net.sourceforge.squirrel_sql

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# SQurreL SQL - Flatpak
```
Comment (barthalion) [inline README.md:1]: In general there's no need to have README as it's unlikely any user will read it.

File: net.sourceforge.squirrel_sql.desktop [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
```
Comment (barthalion) [inline net.sourceforge.squirrel_sql.desktop:1]: Ideally should be upstreamed.

File: net.sourceforge.squirrel_sql.desktop [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+Name=SQuirreL SQL
+Exec=/app/bin/squirrel-sql
```
Comment (barthalion) [inline net.sourceforge.squirrel_sql.desktop:3]:
```suggestion
Exec=squirrel-sql
```

File: net.sourceforge.squirrel_sql.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (barthalion) [inline net.sourceforge.squirrel_sql.appdata.xml:1]: Ideally should be sent and maintained upstream.

**Review by barthalion — COMMENTED** [review]
Just some nitpicks.

**Comment by barthalion** [comment]
Appdata validation fails:

```
builddir/files/share/appdata/net.sourceforge.squirrel_sql.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• style-invalid         : Not enough <screenshot> tags, minimum is 1
```

File: net.sourceforge.squirrel_sql.json [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        "install -Dm644 net.sourceforge.squirrel_sql.desktop /app/share/applications/net.sourceforge.squirrel_sql.desktop"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "squirrel-sql"
+        },
+        {
+          "type": "file",
+          "path": "net.sourceforge.squirrel_sql.appdata.xml"
```
Comment (LeandroStanger) [inline net.sourceforge.squirrel_sql.json:79]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

File: README.md (hunk 1 above)
Comment (ender672) [reply README.md]: I removed the README.

File: net.sourceforge.squirrel_sql.json (hunk 5 above)
Comment (ender672) [reply net.sourceforge.squirrel_sql.json]: I renamed the file to .metainfo.xml

**Comment by nedrichards** [comment]
bot, build net.sourceforge.squirrel_sql

File: net.sourceforge.squirrel_sql.desktop (hunk 2 above)
Comment (ender672) [reply net.sourceforge.squirrel_sql.desktop:1]: I put in a merge request:
https://sourceforge.net/p/squirrel-sql/git/merge-requests/75/

File: net.sourceforge.squirrel_sql.appdata.xml (hunk 4 above)
Comment (ender672) [reply net.sourceforge.squirrel_sql.appdata.xml]: Merge request:
https://sourceforge.net/p/squirrel-sql/git/merge-requests/75/

**Comment by barthalion** [comment]
/merge

Let us know if someone from upstream expresses interest in co-maintaining their app on Flathub.

**Pull request closed without merging**